src/routes/tags.rs

```diff
@@ -302,75 +301,52 @@ pub async fn game_version_delete(
 }
 
 #[derive(serde::Serialize)]
-pub struct LicenseQueryData {
+pub struct License {
     short: String,
     name: String,
 }
 
 #[get("license")]
-pub async fn license_list(
-    pool: web::Data<PgPool>,
-) -> Result<HttpResponse, ApiError> {
-    let results: Vec<LicenseQueryData> = License::list(&**pool)
-        .await?
-        .into_iter()
-        .map(|x| LicenseQueryData {
-            short: x.short,
-            name: x.name,
-        })
-        .collect();
-    Ok(HttpResponse::Ok().json(results))
-}
+pub async fn license_list() -> HttpResponse {
+    let licenses = spdx::identifiers::LICENSES;
+    let mut results: Vec<License> = Vec::with_capacity(licenses.len());
+
+    for (short, name, _) in licenses {
+        results.push(License {
+            short: short.to_string(),
+            name: name.to_string(),
+        });
+    }
 
-#[derive(serde::Deserialize)]
-pub struct LicenseData {
-    name: String,
+    HttpResponse::Ok().json(results)
```

Comment:
in the future: let's remove this from the tags store/fetch in the frontend PR. Doesn't seem like it's needed anymore and this JSON is probs huge